sigstore/sign.py

```diff
@@ -73,7 +73,10 @@
 from sigstore._internal.tuf import TrustUpdater
 from sigstore._utils import sha256_streaming
 from sigstore.transparency import LogEntry
-import newtypes 
+from sigstore._utils import hexstr
+from sigstore._utils import b64str
+from sigstore._utils import pemcert
+from sigstore._utils import keyid
```

Comment:
These should be imported in a single line; `make reformat` should do this for you automatically.